File: challenges/codeeval/medium/013/Skhorn.js
var fs  = require("fs");
var array = [];
var words = "";
var to_erase = [];
fs.readFileSync(process.argv[2]).toString().split('\n').forEach(function (line) {
    if (line !== "") {
        array = line.split(',');
        to_erase = array[1];
        words = array[0];
        for(var i = 1; i <= to_erase.length ; i++)
        {	
        	if(words.indexOf(to_erase[i]) !== -1)
        		words = words.replace(new RegExp(to_erase[i], "g"), '');
        }
        console.log(words);
    }
});

File: challenges/codeeval/easy/103/Skhorn.cs
using System;
using System.IO;
using System.Collections.Generic;
using System.Collections;

class TrailingStrings
{
	public static void Main( string[] args )
	{

		using (StreamReader reader = File.OpenText(args[0]))
		while(!reader.EndOfStream)
		{
			Dictionary<int, int> dict = new Dictionary<int, int>();
			string line = reader.ReadLine();
			if (null == line)
				continue;

			string[] words = line.Split(' ');
			int[] nums = new int[words.Length];
			int[] nums_not_sorted = new int[words.Length];
			int num;
			for(int i = 0; i < words.Length; i++)
			{
				if (Int32.TryParse(words[i], out num))
    				nums[i] = num;
    				nums_not_sorted[i] = num;
			}
			
			Array.Sort(nums);
			foreach(int val in nums)
			{
				if(dict.ContainsKey(val))
					dict[val]++;
				else
					dict[val] = 1;
			}
			int index = 0;
			int flag = 0;
			foreach(var pair in dict)
			{
				if(pair.Value == 1)
				{
					flag = 1;
					index = Array.IndexOf(nums_not_sorted, pair.Key) + 1;
					Console.WriteLine(index);
					break;
				}

			}

			if(flag == 0)
				Console.WriteLine(0);

			flag = 0;
			dict = new Dictionary<int, int>();
		}
	}
}